src/WebJobs.Script/Description/Workers/WorkerFunctionInvoker.cs

```diff
@@ -91,11 +91,23 @@ protected override async Task<object> InvokeCore(object[] parameters, FunctionIn
             };
 
             string invocationId = context.ExecutionContext.InvocationId.ToString();
-            _logger.LogTrace($"Sending invocation id:{invocationId}");
+            ScriptInvocationResult result = new();
+
+            _logger.LogTrace("Sending invocation id: '{id}", invocationId);
             await _functionDispatcher.InvokeAsync(invocationContext);
-            var result = await invocationContext.ResultSource.Task;
+
+            try
+            {
+                result = await invocationContext.ResultSource.Task;
+            }
+            catch (TaskCanceledException ex)
```

Comment:
Should we use the parent type `OperationCanceledException` ? I wonder some APIs we use down the line throws that exception instead of TaskCancelled.

Reply:
I think we should be clear for the specific scenarios we're trying to handle here, but I'm fine to use `OperationCanceledException` instead. My personal preference is to actually check `invocationContext.ResultSource.Task.IsCanceled`